File: Java/program014.java
//Type 2

import java.util.*;

class program14
{
    public static void CheckDivisible(int iNo) 
    {
        if((iNo % 3 == 0) && (iNo % 5 == 0))
        {
            System.out.println("Number is Divisible by 3 and 5");
        }
        else
        {
            System.out.println("Number is Not Divisible by 3 and 5");
        }
    }
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the No: ");
        iValue = sobj.nextInt();
        
        CheckDivisible(iValue);         

        sobj.close();
    }
}

File: Python/program014.py
def CheckEvenOdd(iNo):
    if (iNo % 2) == 0:
        return True
    else:
        return False


iValue = 0
bRet = False

print("Enter number to check whether it is Even or Odd:")
iValue = int(input())

bRet = CheckEvenOdd(iValue)

if bRet == True:
    print(iValue, "is Even")
else:
    print(iValue, "is odd")
